src/test/java/service/DancerServiceImplTest.java

```diff
@@ -0,0 +1,105 @@
+package service;
+
+import helper.BaseTest;
+import java.util.Optional;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import solvd.ermakovich.ct.domain.exception.EntityDoesNotExistException;
+import solvd.ermakovich.ct.domain.node.Dancer;
+import solvd.ermakovich.ct.repository.DancerRepository;
+import solvd.ermakovich.ct.service.impl.DancerServiceImpl;
+
+/**
+ * @author Ermakovich Kseniya
+ */
+@ExtendWith(MockitoExtension.class)
+final class DancerServiceImplTest extends BaseTest {
+
+    @Mock
+    private DancerRepository dancerRepository;
+
+    @InjectMocks
+    private DancerServiceImpl dancerService;
+
+    @Test
+    void createsDancer() {
+        final Dancer dancer = BaseTest.dancer;
+        Mockito.doReturn(dancer)
+                .when(dancerRepository)
+                .save(Mockito.any(Dancer.class));
+        Dancer actualDancer = dancerService.create(dancer);
+        Assertions.assertEquals(
```

Comment:
@kermakovich try to use Hamcrest Matchers instead of Assertions from JUnit package

Reply:
@kermakovich Yes, I have integration tests for neo4j, pr #9

Reply:
@h1alexbel is it a good idea to use a library, that had last release in 2019?

Reply:
@kermakovich what's wrong with the last release in 2019?

hamcrest is the most declarative assertions tool I ever know

Plus, all maven artifacts are immutable and cannot be modified, since if the latest version is working everything is OK

Reply:
@h1alexbel but if smb finds a bug, are there any guarantees, that the bug will be fixed?

Reply:
@kermakovich In such a scenario you can fork hamcrest, fix the bug and maintain the fork

I'm using Hamcrest matchers on a daily basis and happy for now